src/simulator/controllers/map/map_controller.py

```diff
@@ -0,0 +1,52 @@
+from panda3d.core import CollisionTraverser, CollisionHandlerQueue, CollisionNode, BitMask32, CollisionBox, CollisionRay, Point3
+from direct.showbase.ShowBase import ShowBase
+from direct.showbase.DirectObject import DirectObject
+
+from structures import Point
+from simulator.services.debug import DebugLevel
+from simulator.views.map.map_view import MapView
+from simulator.controllers.controller import Controller
+from simulator.controllers.map.cube_map_picker import CubeMapPicker
+from simulator.controllers.map.camera_controller import CameraController
+from simulator.services.event_manager.events.take_screenshot_event import TakeScreenshotEvent
+
+import math
+from typing import Optional
+
+class MapController(Controller, DirectObject):
```

Comment:
a good idea to add more user-instruction comments here or perhaps in the GUI when it comes out